tests/test_basic.py

```diff
@@ -0,0 +1,20 @@
+from collections import OrderedDict
+
+from nose.tools import eq_, assert_true
+
+from mosql.compat import text_type
+from mosql.query import insert
+
+
+def test_simple_insert():
+    mosky = OrderedDict([
+        ('person_id', 'mosky'),
+        ('name', 'Mosky Liu'),
+    ])
+    exp = 'INSERT INTO "person" ("person_id", "name") VALUES (\'mosky\', \'Mosky Liu\')'
+    eq_(insert('person', mosky), exp)
```

Comment:
Yes, `eq_(a, b)` is essentially `assert a == b`. You can supply a third argument to customise the exception message, but it’s not used here.